src/main/java/mybatis/com/mybatis/Repository/StudentRepository.java

```diff
@@ -0,0 +1,18 @@
+package mybatis.com.mybatis.Repository;
+
+import mybatis.com.mybatis.Entity.StudentEntity;
+import mybatis.com.mybatis.Entity.SubjectEntity;
+import org.apache.ibatis.annotations.Param;
+
+import java.util.List;
+
+public interface StudentRepository {
+    List<StudentEntity> getStudents();
```

Comment:
```suggestion
    List<StudentEntity> findAllStudents();
```
Since repos communicate with the database to find/update/save the records the method names should convey that.
We can follow a similar naming convention as JPA.

Reply:
done